ddl/ddl.go

```diff
@@ -497,8 +596,9 @@ func (d *ddl) readyForConcurrencyDDL() {
 			return wk, nil
 		}
 	}
-	d.reorgWorkerPool = newDDLWorkerPool(pools.NewResourcePool(workerFactor(addIdxWorker), batchAddingJobs, batchAddingJobs, 0), reorg)
-	d.generalDDLWorkerPool = newDDLWorkerPool(pools.NewResourcePool(workerFactor(generalWorker), 1, 1, 0), general)
+	d.reorgWorkerPool = newDDLWorkerPool(pools.NewResourcePool(workerFactory(addIdxWorker), reorgWorkerCnt, reorgWorkerCnt, 0), reorg)
```

Comment:
Maybe we can set the initial capacity to `0` if the concurrent DDL is disabled?

Reply:
no, the switch is dynamic, but the initial is only on tidb server startup, the concurrent will not work properly after enable the switch

Reply:
There are initial capacity and max capacity, I mean the initial capacity. Moreover, support controlling the ticker time dynamically can minimize the overhead...